addons/modules/functions/fnc_moduleAmbientAnim.sqf

```diff
@@ -56,17 +61,25 @@ if (isPlayer _unit) exitWith {
         LSTRING(ModuleAmbientAnim_ShowVehicleTheWay),
         LSTRING(ModuleAmbientAnim_TreatWounded),
         LSTRING(ModuleAmbientAnim_CombatWounded),
+        LSTRING(ModuleAmbientAnim_WoundedLeaned),
```

Comment:
Move the leaning,option below genera. Also, change `WoundedLeaned` to `WoundedLeaning`.